src/Opdex.Platform.WebApi/Validation/LiquidityPools/RemoveLiquidityQuoteRequestValidator.cs

```diff
@@ -8,9 +8,16 @@ public class RemoveLiquidityQuoteRequestValidator : AbstractValidator<RemoveLiqu
 {
     public RemoveLiquidityQuoteRequestValidator()
     {
-        RuleFor(request => request.AmountCrsMin).MustBeValidCrsValue().GreaterThan(FixedDecimal.Zero);
-        RuleFor(request => request.AmountSrcMin).MustBeValidSrcValue().GreaterThan(FixedDecimal.Zero);
-        RuleFor(request => request.Liquidity).MustBeValidSrcValue().GreaterThan(FixedDecimal.Zero);
-        RuleFor(request => request.Recipient).MustBeNetworkAddress();
+        RuleFor(request => request.AmountCrsMin)
+            .MustBeValidCrsValue().WithMessage("Minimum CRS amount must contain 8 decimal places.")
+            .GreaterThan(FixedDecimal.Zero).WithMessage("Minimum CRS amount must be greater than 0.");
+        RuleFor(request => request.AmountSrcMin)
+            .MustBeValidSrcValue().WithMessage("Minimum SRC amount must contain 18 decimal places or less.")
+            .GreaterThan(FixedDecimal.Zero).WithMessage("Minimum SRC amount must be greater than 0.");
+        RuleFor(request => request.Liquidity)
+            .MustBeValidSrcValue().WithMessage("Liquidity amount must contain 18 decimal places or less.")
```

Comment:
This can change to specifically look for 8 decimals if you like. Liquidity values will be OLPT tokens which are all 8 decimals.

Reply:
Good idea 👍